oeps/best-practices/oep-0066/Open_edX_Authorization_Systems_Table.rst

```diff
@@ -0,0 +1,199 @@
+   * - **System Role Options**
+     - 
+       * Example roles as named in the db: Administrator, Moderator, etc.
+       * Example roles as named in Instructor Dashboard: Discussion Admin, Discussion Moderator, etc.
+   * - **Example Use Cases**
+     - 
+       * Discussion Service
+   * - **System Details**
+     - 
+       * LMS
+          * Instructor role can access and assign discussion roles
+          * Roles can be assigned in the membership tab on the instructor dashboard page
+          * Roles assigned in the LMS only apply to the LMS Discussions
+          * Administrator role is given to the instructor by default and cannot be removed. Additional administrators can be added 
+   * - **Data Model**
+     - 
+       * All tables are in the edx-platform LMS database
+          * django_comment_client_role table with course_id, id, name fields
+          * django_comment_client_role_users table with id, role_id, user_id, _sdc_deleted_at fields
+          * django_comment_client_permission_roles table with id, role_id, permission_id fields 
+          * django_comment_client_permission table with name field
+
+edx Platform - Django Admin Flow
+---------------------------------
+.. list-table:: 
+   :widths: 15 75
+
+   * - **System Users**
+     - 
+       * Roles are assigned by superusers who have django admin access.
+       * Any user can be assigned a permission set or a group (that has a permission set) through this process.
+   * - **System Role Options**
+     - 
+       * `Currently Existing Roles <https://docs.google.com/document/d/1LZi5r3GYuSlrSRqfVKY1r61ysFQfaVLPJC0p-wVR-3g/edit#heading=h.xe2jv8xthu20>`_
+       
+       *Note: Student is not an explicit role at this time. It is an implicit role based on the existence of an enrollment.*
+   * - **Example Use Cases**
+     - 
+       * Higher level system access
+       * Specific or individualized access required
+       * Higher level of control needed for assigning the roles
+   * - **System Details**
+     - 
+       * Set in the Django Admin Control Panel
+       * users for the instance with superiser role (highest access level) have access to this functionality
+       * poor UX, poor discoverability, overly complex and possibly more error prone 
+
+       *Note: Other services have their own Django Admin Panels that grant permissions.*
+   * - **Data Model**
+     - 
+       * All tables are in the edx-platform LMS database
+          * auth_permission table with codename, content_type_id, id_name fields
+          * auth_group table with id, name fields
+          * auth_group_permissions table with group_id, permission, permission_id, _sdc_deteled_at fields
+          * auth_user table with id, email, username, and many other fields
+          * auth_user_groups table with group_id, id, user_id, _sdc_deleted_at fields
+          * auth_user_user_permissions table with id, permission_id, user_id fields 
+
+edX rbac
```

Comment:
since this is a code package should we call it `edx-rbac` (with dash, no lower case) here and within the following table?